docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Guide to debugging a Sample Ratio Mismatch (SRM)
+description: What to do if you have a Sample Ratio Mismatch (SRM)
+---
+
+You may be visiting this page from the [Help Center Article](https://help.amplitude.com/hc/en-us/articles/8043418569371-Sample-ratio-mismatches-Debug-issues-with-experiment-allocations) or from the [Blog Post](// TODO (AKHIL) add link here once published). A standard question you may have is what do I do that my experiment has an SRM and I understand what an SRM is.
+
+In this guide, we will give a deeper dive into the process of debugging a SRM. 
+
+!!! info
+    This is mostly for the Experiment product, these same steps work for Experiment Results but we do not have information about the assignment event so it is hard to make some of these graphs. Also, we do not know what the intended traffic allocation is. 
+
+## How to debug an SRM
+
+!!! info
+    If you want a TLDR version of this look at the first five items in the list.
+
+Some of this debugging is similar to debugging general software. Is the SRM only on a certain segment of users (i.e. filter by country, OS version, app version, platform, etc). Was there an instrumentation bug that was pushed to a certain app version?
+
+!!! info
+    This list is sorted in rough priority order. 
+
+### 1. Are you using a custom exposure event or the Amplitude exposure event?
+     
+We recommend you use the Amplitude exposure event as this is sent when `fetch()` is called. (TODO AKHIL Should we say something about local event since that doesn't use Amplitude Exposure). On th other hand, if you use a custom exposure event you may not send it at the time the user is exposed to a variant. Note that it does not cost extra money to use Amplitude Exposure events.
```

Comment:
@bgiori Should I write something more here about assignment event being used for local evaluation?

Reply:
I think it'd be good to mention.